### ISSUE TITLE
Any updates of 1.2.3 on maven central

### ISSUE THREAD

**Issue by mbard** [dwdyer/uncommons-maths#10]
Its been quite a a long time since 1.2.3 was release.
And I was hoping you could upload it soon.
It would make our lives a little bit easier :)

**Comment by slandelle** [comment]
Hi there,

We're considering depending on uncommons-maths for our [project](http://github.com/excilys/gatling).
Having 1.2.3 deployed in maven central would be a great help (otherwise, we'll deploy in our own groupId, but that's a pity).

Regards,

Stéphane

**Comment by slandelle** [comment]
Of course, please tell me if I can be of any help in publishing in maven central. This is something I have done countless times.
